pkgs/tools/filesystems/garage/default.nix

```diff
@@ -84,9 +84,9 @@ in
     garage_0_7 = garage_0_7_3;
 
     garage_0_8_2 = generic {
-      version = "0.8.2";
-      sha256 = "sha256-IlDWbNWI1yXvPPF3HIqQvo79M2FQCtoX1wRLJrDbd9k=";
-      cargoSha256 = "sha256-6l4tDBMcOvckTkEO05rman4hHlmVbBt1nCeX5/dETKk=";
+      version = "0.8.3";
+      sha256 = "sha256-NxkFj76L+LpCWzOWbnN3zdhw9Q16uzPibDs+C+voM/0=";
+      cargoSha256 = "sha256-hbBuUjdlw//s6d24dPBu3R/BTJvmOW1B7tSIXNxLXlU=";
     };
 
     garage_0_8 = garage_0_8_2;
```

Comment:
```diff
-    garage_0_8_2 = generic {
+    garage_0_8_3 = generic {
-      version = "0.8.2";
-      sha256 = "sha256-IlDWbNWI1yXvPPF3HIqQvo79M2FQCtoX1wRLJrDbd9k=";
-      cargoSha256 = "sha256-6l4tDBMcOvckTkEO05rman4hHlmVbBt1nCeX5/dETKk=";
+      version = "0.8.3";
+      sha256 = "sha256-NxkFj76L+LpCWzOWbnN3zdhw9Q16uzPibDs+C+voM/0=";
+      cargoSha256 = "sha256-hbBuUjdlw//s6d24dPBu3R/BTJvmOW1B7tSIXNxLXlU=";
    };

-    garage_0_8 = garage_0_8_2;
+    garage_0_8 = garage_0_8_3;
```

or add a separate `garage_0_8_3` definition.

You could also look up an [old version bump](https://github.com/NixOS/nixpkgs/commit/12ebd64d985e674ec55e3d7312c5ecea2c692db2) for this package for inspiration (there is also `all-packages.nix` involved):